backend/webapi/Extensions/TypeExtensions.cs

```diff
@@ -0,0 +1,13 @@
+namespace Pidp.Extensions;
+
+using System.Reflection;
+
+public static class TypetExtensions
```

Comment:
```suggestion
public static class TypeExtensions
```
Typo ?

Reply:
That was indeed a typo. In the newest revision I have removed the extension.